staging/generic-ingress/templates/ingresses.yaml

```
@@ -0,0 +1,21 @@
+{{- range $index, $ingress := .Values.ingresses }}
```

IIRC range in go templates loops through values, so no need for index if not used.

```suggestion
{{- range $ingress := .Values.ingresses }}
```